### ISSUE TITLE
Balances/amounts are displayed like {} when a device's language is not set to English (localization issues)

### ISSUE THREAD

**Issue by elena-zh** [cowprotocol/cowswap#450]
1. Open Cowswap app in a device with any default language but English
2. Verify the app

**AR**: Balances/amounts are displayed like {} when a device's language is not set to English
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/f02fe05e-c0f0-4d20-b5f6-c0becf42e848) ![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/ffa1bf65-a8c8-4ad2-a30b-2411e0138946) ![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/2b06eaf4-a428-44c1-bd29-906f20463d01) ![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/ca8ce140-49e9-4dd9-817b-b30e36267cd0)

**ER**: balances/amount are displayed correctly

**Comment by anxolin** [comment]
@W3stside I believe you removed all the languages but english for now. Could you review if that got lost in the merge?

**Comment by W3stside** [comment]
> 

Yes it did because they moved the logic to a new file name, folder etc
